marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/ModelController.java

```diff
@@ -40,20 +41,23 @@
     @RequestMapping(method = RequestMethod.GET)
     @ResponseBody
     @ApiOperation("This should no longer be used, use /primaryEntityTypes instead")
+    @Secured("ROLE_readEntityModel")
     public ResponseEntity<?> getModels() {
         return ResponseEntity.ok(newModelManager().getModels());
     }
 
     @RequestMapping(value = "/job-info", method = RequestMethod.GET)
     @ResponseBody
     @ApiOperation(value = "Get info about the latest job info for each model", response = LatestJobInfoList.class)
+    @Secured("ROLE_readEntityModel")
     public ResponseEntity<List<JsonNode>> getLatestJobInfoForAllModels() {
         return ResponseEntity.ok(newModelManager().getLatestJobInfoForAllModels());
     }
 
     @RequestMapping(value = "/primaryEntityTypes", method = RequestMethod.GET)
     @ResponseBody
     @ApiOperation(value = "Get primary entity types; does not include entity definitions that are considered 'structured' types", response = PrimaryEntityTypeList.class)
+    @Secured("ROLE_readEntityModel")
     public ResponseEntity<JsonNode> getPrimaryEntityTypes() {
```

Comment:
As being discussed in other threads, we may need to leave the Secured annotation off of this endpoint for the sake of the Curate related roles.